samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -2206,40 +2110,39 @@ mfxStatus CTranscodingPipeline::AllocMVCSeqDesc()
 {
     mfxU32 i;
 
-    m_MVCSeqDesc.View = new mfxMVCViewDependency[m_MVCSeqDesc.NumView];
-    MSDK_CHECK_POINTER(m_MVCSeqDesc.View, MFX_ERR_MEMORY_ALLOC);
-    for (i = 0; i < m_MVCSeqDesc.NumView; ++i)
+    auto mvc = m_mfxDecParams.AddExtBuffer<mfxExtMVCSeqDesc>();
+    mvc->View = new mfxMVCViewDependency[mvc->NumView];
```

Comment:
```
mvc->View = new mfxMVCViewDependency[mvc->NumView]();
```
should zero initialize allocated memory so you won't need to do it explicitly

Reply:
right. thanks.